spec/teaspoon/configuration_spec.rb

```diff
@@ -149,6 +152,19 @@
 
   end
 
+  describe ".coverage_ignored_files" do
+
+    it "returns an empty list if nothing was set" do
+      expect(subject.coverage_ignored_files).to eq([])
+    end
+
+    it "returns an array of  if they were comma separated" do
```

Comment:
Here's another case, but this doesn't feel like a configuration for the CLI, or one that should have an ENV:
https://github.com/modeset/teaspoon/blob/578fd4cd089aede2983e628f1abe60cfcb1a602a/lib/teaspoon/configuration.rb#L138-138